windows/installer.nsi

```diff
@@ -118,8 +118,8 @@ Var bRunApp
     !define prefix "${DEPLOY_PATH}\${buildArch}"
     !tempfile files
 
-    ; Find target folders (Probably here's an issue with quoting. If ${prefix} contains spaces, the installer folders aren't created in the right way.)
-    !system 'cmd.exe /v /c "for /f "usebackq" %d in (`dir /b /s /ad "${prefix}"`) do \
+    ; Find target folders (#864 resolved by adding delims= to the for function. This prevents splitting path names when there is a spaces in the path)
```

Comment:
For isn't a function usually. It's a loop. Can you change the comment? Also there's a grammar mistake: "there is a spaces"

Reply:
You are correct, for is a loop....however cmd help does indicate it is a command. So now fixed the typos and changed function to command.
Thank you for the review!